connectors/v2/trtturku.js

```diff
@@ -35,3 +45,35 @@ Connector.currentTimeSelector = '.jwelapsed';
 Connector.filter = new MetadataFilter({
 	all: [MetadataFilter.trim, toTitleCase]
 });
+
+function setupRadioPlayer() {
+	Connector.artistSelector = '#besa_mp3_play_area tr:nth-child(2) > td:nth-child(2)';
+	Connector.trackSelector = '#besa_mp3_play_area tr:nth-child(3) > td:nth-child(2)';
+}
+
+function setupPlaylistPlayer() {
+	Connector.getArtist = function () {
+		return $('.jwitem.active .jwtitle').text().split(' - ')[0];
+	};
+	
+	Connector.getTrack = function () {
+		return $('.jwitem.active .jwtitle').text().split(' - ')[1];
+	};
```

Comment:
This code can be simplified by using 'Connector.artistTrackSelector' field:
```js
Connector.artistTrackSelector = '.jwitem.active .jwtitle';
```
The core will perform split by itself.